src/app/shared/components/template/components/layout/popup/popup.component.html

```diff
@@ -8,14 +8,18 @@
       <ion-icon slot="icon-only" name="close"></ion-icon>
     </div>
     <div class="popup-content" [attr.data-fullscreen]="props.fullscreen ? true : null">
-      <plh-template-container
-        class="template-container"
-        [name]="props.name"
-        [templatename]="props.templatename"
-        [parent]="props.parent"
-        [row]="props.row"
-        (emittedValue)="handleEmittedValue($event)"
-      ></plh-template-container>
+      @if (props.textOnly) {
+        <div class="large standard normal" [innerHTML]="props.text | markdown"></div>
```

Comment:
`large standard normal` - which is it??! 😅

Reply:
Yeh this is pretty gross... I was using the same pattern to apply those styles as is used by the `text` component, but now I've updated it so that the pop-up component, when called to display text only, uses an instance of the `text` component itself. See https://github.com/IDEMSInternational/open-app-builder/pull/2460/commits/50cc535af47fcd90f96199d1a09b1012b9dff006

The text component styling could still do with some tidying up, but at least this means that it's still only in one place